modules/storage-page-memory/src/main/java/org/apache/ignite/internal/storage/pagememory/AbstractPageMemoryTableStorage.java

```diff
@@ -82,16 +87,20 @@ public TablesConfiguration tablesConfiguration() {
 
     @Override
     public void start() throws StorageException {
-        TableView tableView = tableCfg.value();
-
-        mvPartitions = new AtomicReferenceArray<>(tableView.partitions());
+        inBusyLock(busyLock, () -> {
```

Comment:
Let's reference the busy lock in `inBusyLock()` method using the field directly instead of passing it via a parameter. We have only one busy lock in this object, so we can reference it directly. This will reduce the amount of syntactic noise.

Reply:
Discussed in person.